modules/local/get_prediction_versions.nf

```diff
@@ -10,23 +10,22 @@ process GET_PREDICTION_VERSIONS {
     val external_tool_versions
 
     output:
-    path "versions.yml", emit: versions
+    path "versions.csv", emit: versions
 
     script:
     def external_tools = external_tool_versions.join(",")
 
     """
-    cat <<-END_VERSIONS > versions.yml
-    "${task.process}":
-        mhcflurry: \$(mhcflurry-predict --version 2>&1 | sed 's/^mhcflurry //; s/ .*\$//')
-        mhcnuggets: \$(python -c "import pkg_resources; print('mhcnuggets' + pkg_resources.get_distribution('mhcnuggets').version)" | sed 's/^mhcnuggets//; s/ .*\$//' )
-        epytope: \$(python -c "import pkg_resources; print('epytope' + pkg_resources.get_distribution('epytope').version)" | sed 's/^epytope//; s/ .*\$//')
+    cat <<-END_VERSIONS > versions.csv
+    mhcflurry: \$(mhcflurry-predict --version 2>&1 | sed 's/^mhcflurry //; s/ .*\$//')
+    mhcnuggets: \$(python -c "import pkg_resources; print('mhcnuggets' + pkg_resources.get_distribution('mhcnuggets').version)" | sed 's/^mhcnuggets//; s/ .*\$//' )
+    epytope: \$(python -c "import pkg_resources; print('epytope' + pkg_resources.get_distribution('epytope').version)" | sed 's/^epytope//; s/ .*\$//')
     END_VERSIONS
 
     IFS=',' read -r -a external_tools <<< \"$external_tools\"
     if ! [ -z "${external_tool_versions}" ]; then
         for TOOL in "\${external_tools[@]}"; do
-            echo "\$TOOL" >> versions.yml
+            echo "\$TOOL" >> versions.csv
```

Comment:
maybe add a comment, so we don"t do this again 🙈

Reply:
yes very good point